### ISSUE TITLE
Werte optional für ganz Deutschland

### ISSUE THREAD

**Issue by philoupd** [marcusraitner/COVID-19-Dashboard#29]
Es wäre toll wenn einzelne Werte über die Parameter auch für gesamt Deutschland angezeigt werden könnten.
Ich würde z.B. gerne die Inzidenzwerte standardmäßig für den aktuellen Standort sehen, hätte aber gerne bei den Intensivbetten und insbesondere beim Impfstatus die Werte für Deutschland insgesamt.

**Comment by marcusraitner** [comment]
Impfstatus für DE kann ich nachvollziehen, aber welchen Wert hat die Belegung der Intensivbetten in DE für dich? Ich hätte in der Anzeige auch das Problem, dass die Betten ja unter der Überschrift des Landkreises stehen und wenn ich da einfach DE anzeige, dann verwirrt das vermutlich sehr. Beim Impfstatus ginge das, weil ich dann einfach statt dem Kürzel des Bundeslandes nur "DE" anzeigen könnte.

**Comment by philoupd** [comment]
Es ging mir besonders um den Impfstatus für ganz DE.
Wobei ich es auch hilfreich fände ein zweites Widget anzulegen das alle Werte für DE anzeigt, also auch die Inzidenzen der vergangenen Tage.
Die Belegung der Intensivbetten wären mir dabei aber nicht besonders wichtig.

**Comment by marcusraitner** [comment]
> Es ging mir besonders um den Impfstatus für ganz DE.
> Wobei ich es auch hilfreich fände ein zweites Widget anzulegen das alle Werte für DE anzeigt, also auch die Inzidenzen der vergangenen Tage.

Ja, das könnte Sinn machen; dann wäre es konsistent. Passend dazu gibt es auch den Issue, die Inzidenz-Werte für den Landkreis anzeigen zu lassen (#23 zwar bezogen auf Berlin, wo das auch benötigt wird)

> Die Belegung der Intensivbetten wären mir dabei aber nicht besonders wichtig.

Ok, da sind wir uns einig: das macht eigentlich nur lokal Sinn.

**Comment by vmrv79** [comment]
Da voraussichtlich ab nächster Woche Maßnahmen in Abhängigkeit der Belegung der Intensivbetten getroffen werden (wie ich es verstehe Anzahl Betten in ganz DE, korrigiert mich wenn ich falsch liege), könnte die Anforderung optional die Belegung der Betten für ganz DE anzuzeigen wieder interessant werden.

**Comment by vmrv79** [comment]
Ich glaube es geht bei den Maßnahmen doch nur um die Bettenbelegung auf Bundeslandebene. Dann passt es ja so wie es gerade angezeigt wird.

**Comment by marcusraitner** [comment]
> Ich glaube es geht bei den Maßnahmen doch nur um die Bettenbelegung auf Bundeslandebene. Dann passt es ja so wie es gerade angezeigt wird.

So gesehen ja. Aber eigentlich würde ich früher oder später noch gerne die Werte zu den Hospitalisierten (ggf. als Inzidenz) anzeigen ... aber natürlich gibt es dazu keine API bisher

**Comment by theFerv** [comment]
> Es ging mir besonders um den Impfstatus für ganz DE. **Wobei ich es auch hilfreich fände ein zweites Widget anzulegen das alle Werte für DE anzeigt, also auch die Inzidenzen der vergangenen Tage.** Die Belegung der Intensivbetten wären mir dabei aber nicht besonders wichtig.

Hallo @marcusraitner,

ich würde gerne nochmal dieses Statement von @philoupd pitchen wollen :)  Ein separates Widget für die deutschlandweiten Werte anlegen zu können - quasi als Referenz - fände ich extrem hilfreich. Was meint ihr? Oder gibt es mittlerweile vielleicht schon eine Möglichkeit die Location auf "Deutschland" einzustellen? 

Vielen Dank für das ganze Engagement - ich nutze die Widgets sehr gerne!

**Comment by nils-mac** [comment]
> > Es ging mir besonders um den Impfstatus für ganz DE. **Wobei ich es auch hilfreich fände ein zweites Widget anzulegen das alle Werte für DE anzeigt, also auch die Inzidenzen der vergangenen Tage.** Die Belegung der Intensivbetten wären mir dabei aber nicht besonders wichtig.
> 
> 
> 
> Hallo @marcusraitner,
> 
> 
> 
> ich würde gerne nochmal dieses Statement von @philoupd pitchen wollen :)  Ein separates Widget für die deutschlandweiten Werte anlegen zu können - quasi als Referenz - fände ich extrem hilfreich. Was meint ihr? Oder gibt es mittlerweile vielleicht schon eine Möglichkeit die Location auf "Deutschland" einzustellen? 
> 
> 
> 
> Vielen Dank für das ganze Engagement - ich nutze die Widgets sehr gerne!

Hallo
Ich gände es auch sehr schön, wenn  es ein Medium-Widget gäbe mit Werten nur für Deutschland. Also Inzidenz (inkl. Verlauf und Neue Infektionen),
Impfquote (gerne auch mit einer Nachkommastelle) und Hospitalisiertungsinzidenz. 
Gibt es da eine Möglichkeit für oder müsste man ein komplett eigenes Skript schreiben?
LG

**Comment by marcusraitner** [comment]
> > > Es ging mir besonders um den Impfstatus für ganz DE. **Wobei ich es auch hilfreich fände ein zweites Widget anzulegen das alle Werte für DE anzeigt, also auch die Inzidenzen der vergangenen Tage.** Die Belegung der Intensivbetten wären mir dabei aber nicht besonders wichtig.
> > 
> > 
> > Hallo @marcusraitner,
> > ich würde gerne nochmal dieses Statement von @philoupd pitchen wollen :)  Ein separates Widget für die deutschlandweiten Werte anlegen zu können - quasi als Referenz - fände ich extrem hilfreich. Was meint ihr? Oder gibt es mittlerweile vielleicht schon eine Möglichkeit die Location auf "Deutschland" einzustellen?
> > Vielen Dank für das ganze Engagement - ich nutze die Widgets sehr gerne!
> 
> Hallo Ich gände es auch sehr schön, wenn es ein Medium-Widget gäbe mit Werten nur für Deutschland. Also Inzidenz (inkl. Verlauf und Neue Infektionen), Impfquote (gerne auch mit einer Nachkommastelle) und Hospitalisiertungsinzidenz. Gibt es da eine Möglichkeit für oder müsste man ein komplett eigenes Skript schreiben? LG

Machbar ist das. Alle Werte (Inzidenz, Impfquote und Hospitalisierung sind bei `https://api.corona-zahlen.org/` abrufbar. Im Prinzip kann man dann schon die Logik meines Widget nehmen und daraus ein neues bauen. Es in dieses mit einzubauen ist vermutlich nicht zielführend. Da muss ich erst mal drüber schlafen.
